tests/index.html

```diff
@@ -40,9 +40,7 @@
         window.Ember = {
           testing: true
         };
-        window.ENV = window.ENV || {
-          _ENABLE_LEGACY_VIEW_SUPPORT: true
```

Comment:
The tests still use this flag. A number of tests still use the {{view helper. A great many of the usages of {{view have been replaced during the migration to ember-glimmer, but many still exist (in other packages than ember-htmlbars).